orienteer-core/src/main/java/org/orienteer/core/component/command/DeleteODocumentCommand.java

```diff
@@ -49,10 +36,11 @@ public DeleteODocumentCommand(OrienteerDataTable<ODocument, ?> table, IModel<OCl
 	@Override
 	protected void performMultiAction(AjaxRequestTarget target, List<ODocument> objects) {
 		super.performMultiAction(target, objects);
-		ODatabaseDocument db = getDatabase();
-		db.commit(true);
-		db.begin();
-		db.getMetadata().reload();
+		DBClosure.sudoConsumer(db -> {
+			db.commit(true);
+			db.begin();
+			db.getMetadata().reload();
```

Comment:
Do you understand why reload of metadata needed here? Probably because of deletion of Ofunction and etc., but not sure.

Reply:
You added reload metadata in commit 9903412516235a12bb4e2dab1a16a62ee4fd75d

https://github.com/OrienteerBAP/Orienteer/commit/9903412516235a12bb4e2dab1a16a62ee4fd75d0#diff-6a4d849e457c288ccfc76d57760b719e